sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -966,6 +978,9 @@ private[hive] class HiveClientImpl(
 }
 
 private[hive] object HiveClientImpl {
+  val DB_OWNER_NAME_PROP: String = "ownerName"
+  val DB_OWNER_TYPE_PROP: String = "ownerType"
```

Comment:
nit: we should define these special properties in `SupportsNamespaces`

Reply:
Hmm..  How can `SupportsNamespaces` work with tables?  these properties are common for tables and dbs.

Reply:
It seems that we already duplicate some props, I will follow this.